requirements3.txt

```diff
@@ -0,0 +1,6 @@
+requests >= 2.27.1
+PyYAML >= 5.4.1
+pynuodb >= 2.5.1
+#pynuoadmin >= 2.0.0
```

Comment:
Why?

Reply:
the latest pynuoadmin should be a requirement but it's not available on pypi.